gradle.properties

```diff
@@ -1,10 +1,10 @@
 # Sets default memory used for gradle commands. Can be overridden by user or command line properties.
 # This is required to provide enough memory for the Minecraft decompilation process.
 org.gradle.jvmargs=-Xmx3G
-mod_version=3.1.0
+mod_version=4.0.1
```

Comment:
If your still following my versioning, then 1.12 should have been 7.x

| MC | GC |
| --- | --- |
| 1.7 | 2.x |
| 1.8 | 3.x |
| 1.9 | 4.x |
| 1.10 | 5.x |
| 1.11 | 6.x | 
| 1.12 | 7.x |

Using a different major version for GC for a different version of minecraft means users would get less confused (which GC version works with which version of MC)

Reply:
That versioning system hasn't been used for quite a few MC versions.

https://github.com/GrowthcraftCE/Growthcraft-1.12/blob/development/build.properties

https://github.com/GrowthcraftCE/Growthcraft-1.11/blob/development/gradle.properties

https://github.com/GrowthcraftCE/Growthcraft-1.10/blob/development/gradle.properties

Reply:
Yeah, looks like they forgot to bump it with each version